Update README and tutorial (#118)
wagtail/wagtail-live: README.md

![Wagtail Live: Page edit view](https://wagtail.github.io/wagtail-live/images/wagtail-admin.jpg)

## Compatibility

Wagtail Live works with:

- Wagtail 2.12 or higher
- Django 2.2 or higher
- Python 3.7 or higher

## Getting started

To get started with Wagtail Live, check out the [Wagtail Live tutorial](https://wagtail.github.io/wagtail-live/getting_started/tutorial/).

## Documentation

All documentation is in the "docs" directory and online at [wagtail.github.io/wagtail-live/](https://wagtail.github.io/wagtail-live/).

## Community

> We have a `wagtail-live` channel in [Wagtail's Slack workspace](https://github.com/wagtail/wagtail/wiki/Slack) where happens discussions about the project.

Reviewer: ```suggestion
We have a `wagtail-live` channel in [Wagtail's Slack workspace](https://github.com/wagtail/wagtail/wiki/Slack) for discussions about the project.
```
